tests/10-create.js

```diff
@@ -0,0 +1,160 @@
+/*!
+ * Copyright 2023 Digital Bazaar, Inc. All Rights Reserved
+ */
+import {shouldBeBs58, shouldBeMulticodecEncoded} from './assertions.js';
+import chai from 'chai';
+import {
+  checkDataIntegrityProofFormat
+} from 'data-integrity-test-suite-assertion';
+import {createInitialVc} from './helpers.js';
+import {documentLoader} from './documentLoader.js';
+import {endpoints} from 'vc-api-test-suite-implementations';
+import {validVc as vc} from './validVc.js';
+
+const tag = 'ecdsa-2019';
+const cryptosuite = 'ecdsa-2019';
+const {match, nonMatch} = endpoints.filterByTag({
+  tags: [tag],
+  property: 'issuers'
+});
+const should = chai.should();
+
+describe('ecdsa-2019 (P-256 create)', function() {
+  checkDataIntegrityProofFormat({
+    implemented: match,
+    notImplemented: nonMatch
+  });
+  describe('ecdsa-2019 (P-256 issuer)', function() {
+    this.matrix = true;
+    this.report = true;
+    this.implemented = [...match.keys()];
+    this.notImplemented = [...nonMatch.keys()];
+    this.rowLabel = 'Test Name';
+    this.columnLabel = 'Implementation';
+    for(const [columnId, {endpoints, implementation}] of match) {
+      const [issuer] = endpoints;
+      const verifier = implementation.verifiers.find(
+        v => v.tags.has(tag));
+      let issuedVc;
+      let proofs;
+      let verificationMethodDocuments;
+      before(async function() {
+        issuedVc = await createInitialVc({issuer, vc});
+        proofs = Array.isArray(issuedVc?.proof) ?
+          issuedVc.proof.filter(proof => proof.cryptosuite === cryptosuite) :
+          [issuedVc?.proof];
+        proofs.should.not.eql([], 'Expected at least one proof to have ' +
+          'cryptosuite "ecdsa-2019".');
+        const verificationMethods = proofs.map(
+          proof => proof.verificationMethod);
+        verificationMethodDocuments = await Promise.all(
+          verificationMethods.map(async verificationMethod => {
+            const verificationMethodDocument = await documentLoader({
```

Comment:
NOTE: The test suite is set up to use did:key and http/https URLs as the verification method. Doesn't support other types.

Should it be able to resolve other DID types?

Reply:
if it is required we can probably ask the vendor to stand up a did resolver that returns a did document for their did or add another did method driver to the documentLoader. Good question and thanks for the code.

Reply:
No, did:key is fine for now.